app/routes/session.py

```diff
@@ -163,12 +165,11 @@ def get_sign_out():
     """
     survey_config = get_survey_config()
     log_out_url = (
-        survey_config.account_service_todo_url if "todo" in request.args else None
+        url_for("session.get_signed_out")
+        if "external_redirect" in request.args
+        else survey_config.account_service_log_out_url
```

Comment:
This needs to be inverse. Or you need to use `internal_redirect`.
```suggestion
        survey_config.account_service_log_out_url
        if "external_redirect" in request.args
        else url_for("session.get_signed_out")
```

Reply:
That makes more sense now. I was confused at why you wanted it to be `external` because I was keeping the logic the same as the `save` param. Ill change it to internal